SpaceHosting.Index.Tests/IndexStoreTests.cs

```diff
@@ -11,7 +11,7 @@ public class IndexStoreTests
     {
         private const double singlePrecisionEpsilon = 1e-06;
 
-        private readonly Random random = new Random();
```

Comment:
I personally am not a big fan of this syntax sugar, but it's all very subjective obviously...